test/bucket_tests.cpp

```diff
@@ -86,69 +86,66 @@ BOOST_AUTO_TEST_CASE( delimiter ) {
     free_client(client);
 }
 
-BOOST_AUTO_TEST_CASE(marker){
-	ds3_request* request;
-	ds3_error* error;
-	ds3_get_bucket_response* response;
-	ds3_client* client = get_client();
-	const char* bucket_name = "bucket_test_marker";
-	uint64_t num_objs;
-	
-	populate_with_objects(client,bucket_name);
-	
-	request = ds3_init_get_bucket(bucket_name);
-	ds3_request_set_marker(request,"resources/sherlock_holmes.txt");
-	error = ds3_get_bucket(client,request,&response);
-	ds3_free_request(request);
-	
-	handle_error(error);
-	num_objs = response->num_objects;
-	
-	BOOST_CHECK_EQUAL(num_objs, 3);
-	BOOST_CHECK(contains_object(response->objects, num_objs, "resources/tale_of_two_cities.txt"));
-	BOOST_CHECK(contains_object(response->objects, num_objs, "resources/ulysses.txt"));
-	BOOST_CHECK(contains_object(response->objects, num_objs, "resources/ulysses_large.txt"));
-	
-	ds3_free_bucket_response(response);
-
-  clear_bucket(client, bucket_name);
-  free_client(client);
+BOOST_AUTO_TEST_CASE(marker) {
+	  ds3_request* request;
+	  ds3_error* error;
+	  ds3_get_bucket_response* response;
+	  ds3_client* client = get_client();
+	  const char* bucket_name = "bucket_test_marker";
+	  uint64_t num_objs;
+
+	  populate_with_objects(client,bucket_name);
+
+	  request = ds3_init_get_bucket(bucket_name);
+	  ds3_request_set_marker(request,"resources/sherlock_holmes.txt");
+	  error = ds3_get_bucket(client,request,&response);
+	  ds3_free_request(request);
+
+	  handle_error(error);
+	  num_objs = response->num_objects;
+
+	  BOOST_CHECK_EQUAL(num_objs, 3);
+	  BOOST_CHECK(contains_object(response->objects, num_objs, "resources/tale_of_two_cities.txt"));
+	  BOOST_CHECK(contains_object(response->objects, num_objs, "resources/ulysses.txt"));
+	  BOOST_CHECK(contains_object(response->objects, num_objs, "resources/ulysses_large.txt"));
+
+	  ds3_free_bucket_response(response);
+
+    clear_bucket(client, bucket_name);
+    free_client(client);
 }
 
-BOOST_AUTO_TEST_CASE(max_keys)
-{
-	ds3_request* request;
-	ds3_error* error;
-	ds3_get_bucket_response* response;
-	ds3_client* client = get_client();
-	const char* bucket_name = "bucket_test_max_keys";
-	uint64_t num_objs;
-	
-	populate_with_objects(client,bucket_name);
-	
-	request = ds3_init_get_bucket(bucket_name);
-	ds3_request_set_max_keys(request,2);
-	
-	error = ds3_get_bucket(client,request,&response);
-	ds3_free_request(request);
-	
-	handle_error(error);
-	num_objs = response->num_objects;
-	
-	BOOST_CHECK_EQUAL(num_objs, 2);
-	BOOST_CHECK(contains_object(response->objects, num_objs, "resources/beowulf.txt"));
-	BOOST_CHECK(contains_object(response->objects, num_objs, "resources/sherlock_holmes.txt"));
-	ds3_free_bucket_response(response);
+BOOST_AUTO_TEST_CASE(max_keys) {
+    ds3_request* request;
+	  ds3_error* error;
```

Comment:
Oh, I just need to add more spaces.  My default tab settings are set to 2 instead of 4.